app/views/categories/index.html.erb

```diff
@@ -0,0 +1,107 @@
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, :back, class: "btn" %>
+  <h1>Categories</h1>
+  <%= form_for(current_user, url: destroy_user_session_path, method: :delete, html: { class: "nav-link" }) do |f| %>
+    <%= f.button type: :submit, class: "btn-logout" do %>
+      <%= content_tag :i, "", class: "fa-solid fa-right-from-bracket" %>
+    <% end %>
+  <% end %>
+</div>
+<% if @categories.any? %>
+  <ul class="categories">
+    <% @categories.each do |category| %>
+      <li class="category">
+      <div class="info">
+        <% if category.icon.present? %>
+          <%= raw category.icon %>
+        <% end %>
+        <h3><%= category.name %>:</h3>
+        <p>$<%= category.expenses.collect(&:amount).sum %></p>
+    </div>
+    <div class="btns">
+        <div class="btn">
+          <button><a href="<%= category_expenses_path(category) %>">View Expenses</a></button>
+          <button><%= link_to 'Add a New Expense', new_category_expense_path(category) %></button>
+        </div>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <button class="btn addNew"><%= link_to 'Add a New Category', new_category_path %></button>
```

Comment:
- [ ] As pointed out by the previous reviewer, you'll be required to work once more on your design. Based on the requirements of this project, this button should be right at the bottom of the page. Once more kindly check out the original design guidelines. Composition and space between elements should be the same as in the original design. Kindly adhere to the project requirements and make sure your app is very responsive.

| Design guidelines :+1:  | Yours |
| --- | --- |
| ![Screenshot from 2023-05-20 11-30-02](https://github.com/Salwa99/My-Budget/assets/63087491/135ed99d-1a7d-484b-a443-67ec0b0e170d) | ![Screenshot from 2023-05-20 11-29-15](https://github.com/Salwa99/My-Budget/assets/63087491/13d401b5-2f82-443c-9b34-187b46e22fdf) |